### PR TITLE
Entities and endpoints

### PR DESCRIPTION
**Pull request by mmathias**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,27 +1,63 @@
-from flask import Flask, request
 from database import db
-import user
```
Comment (mmathias) [inline app.py:3]: so we don't need that, right?

File: app.py [hunk 2]
```diff
@@ -1,27 +1,63 @@
...
+from flask import Flask, request
+from models import User, Business, Point
```
Comment (mmathias) [inline app.py:3]: When we import it we get access directly to the db operations?

File: app.py [hunk 3]
```diff
@@ -1,27 +1,63 @@
...
+    return "Hello Fidelity!"
+
 
 @app.route("/users", methods=["POST"])
 def create_user():
     email = request.form["email"]
 
-    # work with sess
-    new_user = user.User(email)
+    new_user = User(email)
```
Comment (mmathias) [inline app.py:20]: Interesting!

File: app.py [hunk 4]
```diff
@@ -1,27 +1,63 @@
...
     db.session.add(new_user)
     db.session.commit()
 
     return "Created User! %s " % email
 
 
+@app.route("/users/<int:id>", methods=["GET"])
+def get_user(id):
+    user = User.query.get(id)
```
Comment (mmathias) [inline app.py:29]: don't you have something similar for saving or updating?

File: models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+from .user import User
```
Comment (mmathias) [inline models/__init__.py:1]: what are these about?

File: models/point.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    user_id = Column(Integer, ForeignKey('users.id'))
+    business_id = Column(Integer, ForeignKey('business.id'))
+    active_round = Column(Boolean, default=True, nullable=False)
+
+    def __init__(self, user_id, business_id):
+        self.user_id = user_id
+        self.business_id = business_id
+
+    def __repr__(self):
+        return "<Point(id='%s')>" % self.id
```
Comment (mmathias) [inline models/point.py:17]: nice =D

File: app.py (hunk 1 above)
Comment (IgnasiBosch) [inline app.py:3]: in line 3 we import classes directly.
in file models/**init**.py you can appreciate that we make all imports as a module. So in models we can access to classes

File: models/point.py (hunk 6 above)
Comment (mmathias) [inline models/point.py:17]: this here is just a toString(), right?

File: app.py (hunk 2 above)
Comment (IgnasiBosch) [inline app.py:3]: Yes, all that you import is like have it in your current file

File: app.py (hunk 3 above)
Comment (IgnasiBosch) [inline app.py:20]: This is the point. A little cleaner

File: app.py (hunk 1 above)
Comment (mmathias) [inline app.py:3]: as a module? What do you mean?

File: app.py (hunk 4 above)
Comment (IgnasiBosch) [inline app.py:29]: It's pretty similar but not exactly the same. SqlAlchemy is a data mapper, we use db.session for that:
To get we use a static method "User.query..." is not an instance, is a class.
To save it:
user_instance = User
user_instance.name = 'Mickey Mouse'
db.session.add(user_instance)
db.session.commit()  <--- Boom, is in db

Comment (mmathias) [inline app.py:29]: got it, a bit ugly hehe. But it is fine =D

File: models/point.py (hunk 6 above)
Comment (IgnasiBosch) [inline models/point.py:17]: Yes, a kind of. Is like an internal-human-readable id

File: models/__init__.py (hunk 5 above)
Comment (IgnasiBosch) [inline models/__init__.py:1]: We are making User class accessible directly from models module:
models.User

**Pull request merged**